jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1256,10 +1256,23 @@ void java_bytecode_parsert::rmethod_attribute(methodt &method)
           id2string(method.name)+":"+method.descriptor);
     }
 
-    // line number of method
+    // line number of method (the first line number available)
     if(!method.instructions.empty())
```

Comment:
Isn't that covered by the check in line 1268?

Reply:
Indeed, fixed